CONFIGURATION.md

```diff
@@ -100,7 +100,7 @@ queue.buffering.max.ms                   |  P  | 0 .. 900000     |             0
 linger.ms                                |  P  |                 |               | Alias for `queue.buffering.max.ms`
 message.send.max.retries                 |  P  | 0 .. 10000000   |             2 | How many times to retry sending a failing MessageSet. **Note:** retrying may cause reordering. <br>*Type: integer*
 retries                                  |  P  |                 |               | Alias for `message.send.max.retries`
-retry.backoff.ms                         |  P  | 1 .. 300000     |           100 | The backoff time in milliseconds before retrying a protocol request. <br>*Type: integer*
+retry.backoff.ms                         |  P  | 1 .. 300000     |           100 | The backoff time in milliseconds before retrying a protocol request.<br>*Type: integer*
```

Comment:
CONFIGURATION.md is automatically generated on build from rdkafka_conf.c, so you'll need to update that file, rerun 'make', and then commit both files.